src/Menu/MenuManager.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace FluxErp\Menu;
+
+use FluxErp\Models\Permission;
+use Illuminate\Routing\Route;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+use Illuminate\Support\Traits\Macroable;
+use Spatie\Permission\Exceptions\PermissionDoesNotExist;
+
+/**
```

Comment:
DocBlock can be removed, this is not the facade, this is the concrete implementation